conf/examples/provider_mode_ibm_cloud_baremetal_kubevirt_clients.yaml

```diff
@@ -0,0 +1,36 @@
+# This yaml file contains the configuration for the deployment of the ACM hub cluster on IBM Cloud Bare Metal
+# with Hosted client clusters. The configuration is used by the HyperShiftBase.download_hcp_binary(),
+# HostedClients.deploy_hosted_ocp_clusters(), HypershiftHostedOCP.deploy_ocp(), HostedODF.do_deploy(),
+# MetallbOperator.deploy_metallb() and similar methods during the Hosted clusters deployment.
+# The configuration is for observation purposes only and is not intended to be executed. In order to execute
+# the configuration, the user must provide the desired values for the variables in the configuration.
+
```

Comment:
What about renaming this file and adding suffix `.example` (similarly to `conf/ocsci/vsphere_upi_vars.yaml.example`)?